app/models/like.rb

```diff
@@ -0,0 +1,7 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  def update_likes_counter
+    post.update(likes_counter: user.likes.count)
+  end
```

Comment:
- [ ] It would be nice to use the `after_save` to call the `#update_likes_counter` method every time a new comment has been saved to database, then make the `update_likes_counter` method private, just like in the post model, check the post model to see the suggestion 👍🏿

Reply:
This change has not yet been implemented

Reply:
- [ ] Just like the previous reviewer has said  It would be nice to use the` after_save` to call the `#update_likes_counter `method every time a new comment has been saved to database, then make the `update_likes_counter` method private, just like in the post model